package/yast2-packager.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Wed Jul  6 10:46:54 UTC 2016 - igonzalezsosa@suse.com
+
+- Fix remaining packages count during installation (bsc#987791)
```

Comment:
Yes, I discovered that the second bug was fixed after creating the PR :) Updated now.